server_cmds_test.go

```diff
@@ -306,19 +306,87 @@ func testCmdsServer(t *testing.T, c *Client) {
 
 		expected := []*OnlineClient{
 			{
-				ID:          5,
-				ChannelID:   7,
-				DatabaseID:  40,
-				Nickname:    "ScP",
-				Type:        0,
-				Away:        true,
-				AwayMessage: "not here",
+				ID:         42087,
+				ChannelID:  39,
+				DatabaseID: 19,
+				Nickname:   "bdeb1337",
+				Type:       0,
+				OnlineClientAway: OnlineClientAway{
+					Away:        false,
+					AwayMessage: "",
+				},
 			},
 		}
 
 		assert.Equal(t, expected, clients)
 	}
 
+	clientlistextended := func(t *testing.T) {
+		t.Helper()
+		clientz, err := c.Server.ClientList(ClientListFull)
+		if !assert.NoError(t, err) {
+			return
+		}
+
+		// helper functions to return pointers
+		boolptr := func(b bool) *bool {
+			return &b
+		}
+		stringptr := func(s string) *string {
+			return &s
+		}
+		intptr := func(i int) *int {
+			return &i
+		}
```

Comment:
With the change to pointer embeds these might not be needed.

Reply:
I think they are still needed, except for the boolptr function, I have made variables for this. Unless there would be another way that you think is better?